src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -767,6 +769,18 @@ void testAcceptHandlesExceptionsProperly() {
             assertThrows(IllegalArgumentException.class, () -> watch.accept(consumer).accept("a", "b"));
             assertTrue(watch.isSuspended());
         }
+
+        @Test
```

Comment:
@garydgregory That is the way I would prefer to ensure that the time is really taken. WDYT?

Reply:
Hm, this assumes that `start()` and `resume()` actually work and does not actually "tests that time is really taken" (`testTimeIsReallyTaken()`), since no time is checked. To me the test name and implementation say different things.